### PR TITLE
Input component

### PR DESCRIPTION
**Pull request by renancarvalho**
Add input component

### CODE REVIEW

File: src/components/ui-input.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<template>
+    <div class="ui-container">
```
Comment (dine) [inline src/components/ui-input.vue:2]: I think we should replace this class for `ui-relative`
For when we need to add prefix or suffix to input

File: src/components/ui-input.vue [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <input
+            :type="type"
+            :value="value"
+            :class="isValid !== true ? 'ui-border-red-1' : 'ui-border-light-1'"
+            class="ui-container ui-rounded-sm ui-rounded-sm ui-border ui-bg-white ui-shadow-sm ui-p-2 ui-pl-4 ui-outline-none ui-text-base ui-font-normal"
+            :placeholder="placeHolder"
+            v-bind="$attrs"
+            v-on="$listeners"
+        />
+        <p v-if="!isValid" class="ui-text-red-1 ui-mt-1">
```
Comment (dine) [inline src/components/ui-input.vue:12]: I think this should be another component, like `ui-error-message`

File: src/components/ui-input.vue [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<template>
+    <div class="ui-container">
+        <input
+            :type="type"
+            :value="value"
```
Comment (dine) [inline src/components/ui-input.vue:5]: If you are binding attributes I don't think we need this line

File: src/components/ui-input.vue [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            :class="isValid !== true ? 'ui-border-red-1' : 'ui-border-light-1'"
+            class="ui-container ui-rounded-sm ui-rounded-sm ui-border ui-bg-white ui-shadow-sm ui-p-2 ui-pl-4 ui-outline-none ui-text-base ui-font-normal"
+            :placeholder="placeHolder"
```
Comment (dine) [inline src/components/ui-input.vue:8]: Same comment as before

File: src/components/ui-input.vue (hunk 2 above)
Comment (renancarvalho) [reply src/components/ui-input.vue:12]: Indeed! added to PR too!

File: src/components/ui-input.vue (hunk 3 above)
Comment (renancarvalho) [reply src/components/ui-input.vue:5]: "Type" didnt work without.
"Value: is gone

File: src/components/ui-input.vue (hunk 4 above)
Comment (renancarvalho) [reply src/components/ui-input.vue:8]: Placeholder must be there event with the v-bind...

File: src/components/ui-input.vue (hunk 1 above)
Comment (renancarvalho) [reply src/components/ui-input.vue:2]: Think we need both  =)

**Review by dine — APPROVED** [review]

**Pull request merged**